web/.env

```diff
@@ -0,0 +1 @@
+VITE_API_URL="http://localhost:3000"
```

Comment:
The `.env` file with all of your secrets should never be pushed to the repository, it should be included in the `.gitignore` file.

Note: Of course, the `.env` template can be posted since that provides information on what variables should be set in order to run your project.